src/modules/launcher/Wox.Plugin/PluginMetadata.cs

```diff
@@ -57,6 +57,8 @@ public override string ToString()
             return Name;
         }
 
+        public bool DelayedExecution { get; set; } = false;
```

Comment:
[clarification] Is this required ? `QueryForPlugin` would give an empty result and we already have a check later for empty results.

Reply:
Currently QueryForPlugin will still perform the query if you pass in `true` for the last argument and it is not an `IDelayedExecutionPlugin`. Should I change it such that if `true` is passed it will only run the query for plugins which implement `IDelayedExecutionPlugin`? This would mean that there would be more calls to QueryForPlugin though.

Reply:
IMO it would be better to query depending on `delayedExecution` value. This would prevent explicit need to modify metadata to run long queries.

Reply:
Removed the metadata from the json file and from code and modified `QueryForPlugin` accordingly.